build.zig

```diff
@@ -421,6 +426,7 @@ pub fn jetzigInit(b: *std.Build, exe: *std.Build.Step.Compile, options: JetzigIn
 fn registerDatabaseSteps(b: *std.Build, exe_database: *std.Build.Step.Compile) void {
     const commands = .{
         .{ "migrate", "Migrate your Jetzig app's database." },
+        .{ "seed", "Run seeds and setup initial data." },
```

Comment:
"set up" instead of "setup" ("set up" is a verb, "setup" is a noun).

Reply:
Done!

Reply:
Nice, so now just JetQuery PR to come and then update this PR with the new version once it's merged, then we're all done.

Let me know if you need any help !